modules/nodes-secondary-scheduler-about.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/index.adoc
+
+:_content-type: CONCEPT
+[id="nodes-secondary-scheduler-about_{context}"]
+= About the {secondary-scheduler-operator}
```

Comment:
@ingvagabund @kasturinarra I couldn't find much to say about the secondary scheduler. Let me know if you can think of anything else to say, or if there are any other resources I should look at to pull info from.

Please take a look closely at this section and see if there are any clarifications to the wording I've used or if I'm saying anything wrong.

Reply:
Worth mentioning a paragraph about expectations about the scheduler image. We [silently assume](https://github.com/openshift/secondary-scheduler-operator/blob/master/bindata/assets/secondary-scheduler/deployment.yaml#L37-L38) every image has `/bin/kube-scheduler` binary. The image building process as not important as long as the binary is present and it is based on the scheduling framework.

Reply:
@ingvagabund thanks! I incorporated this here, let me know if this is okay or not:

```
The custom scheduler must have the `/bin/kube-scheduler` binary and be based on the link:https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/[Kubernetes scheduling framework].
```

Reply:
Looks good